<?xml version="1.0"?>
<launch>

	<rosparam command="load" file="$(find ar_navigation)/qr_positions/qr_positions.yaml"/>

	<!-- qr_saver node -->
	<node pkg="ar_navigation" type="qr_localization" name="qr_localization" output="screen"/>
	
</launch>

